app/package.json

```diff
@@ -52,6 +52,7 @@
     "eject": "react-scripts eject",
     "serve": "serve -s build"
   },
+  "homepage": ".",
```

Comment:
according to the docs this should allow us to serve the app at a different path (needed for dpeloyment, where it will be at "http://..../packit") https://create-react-app.dev/docs/deployment/#serving-the-same-build-from-different-paths